model/src/main/kotlin/PackageCuration.kt

```diff
@@ -65,10 +64,7 @@ data class PackageCuration(
      */
     private fun isApplicableIvyVersion(pkgId: Identifier) =
         runCatching {
-            // TODO: This check does not completely comply to the Ivy version-matchers specification. E.g. the version
-            //       '1.0' does not satisfy the version range [1.0,2.0], see
-            //       https://github.com/vdurmont/semver4j/issues/67.
-            Requirement.buildIvy(id.version).isSatisfiedBy(Semver(pkgId.version, Semver.SemverType.LOOSE))
+            Semver.coerce(pkgId.version).satisfies(id.version)
```

Comment:
`satisfies(String)` never uses `coerce` internally.
Isn't this required to solve the `TODO` above?

Reply:
It shouldn't be necessary (anymore), see https://github.com/semver4j/semver4j/issues/100. And I just manually verified that `Semver.coerce("1.0").satisfies("[1.0,2.0]") == true`.